### PR TITLE
NOREF better bulk id posting

### PR DESCRIPTION
**Pull request by nonword**
Adds support for processing a CSV of prefixed ids from multiple institutions. E.g.

```
b1234
cb5678
...
```

### CODE REVIEW

File: run.js [hunk 1]
```diff
@@ -41,6 +44,23 @@ function promptWithDefault (question, defaultValue) {
   })
 }
 
+function nyplSourceAwareArrayChunks (arr, chunkSize = 25) {
```
Comment (charmingduchess) [inline run.js:43]: can you add comments describing whats happening here?

File: run.js [hunk 2]
```diff
@@ -82,25 +105,39 @@ function runWithOptions () {
   }
 }
 
+let csvIndex = argv.offset || 0
 let csvChunkIndex = 0
 function runOverCsv (chunks) {
   ids = chunks[csvChunkIndex]
 
   if (ids && ids.length > 0) {
```
Comment (charmingduchess) [inline run.js:126]: so chunks here would be the sorted groups from nyplSourceAwareChunks?

Comment (charmingduchess) [reply run.js:126]: If so, can you leave a comment describing why we're just checking the first id?

**Review by charmingduchess — APPROVED** [review]

File: run.js (hunk 2 above)
Comment (nonword) [reply run.js:126]: Added additional inline comments (and slight refactoring) to try to clarify things

File: run.js (hunk 1 above)
Comment (nonword) [reply run.js:43]: Added function documentation.

**Pull request merged**